tests/Export/ProductIdExportTest.php

```diff
@@ -6,17 +6,14 @@
 
 use FINDOLOGIC\FinSearch\Export\ProductIdExport;
 use FINDOLOGIC\FinSearch\Export\XmlExport;
+use Shopware\Core\Content\Category\CategoryEntity;
+use Shopware\Core\Content\Product\ProductEntity;
 use Shopware\Storefront\Framework\Routing\Router;
 
 class ProductIdExportTest extends XmlExportTest
 {
-    public function testProductsInCrossSellCategoriesAreNotWrappedAndErrorIsLogged(): void
+    public function buildItemsAndAssertError(ProductEntity $product, CategoryEntity $category)
```

Comment:
This method is never called, therefore nothing tests this logic for the product id export.

Reply:
The test class extends `XmlExportTest`, so this is just an extended version of the helper function defined there.
It is called during the tests of `XmlExportTest`.

![image](https://user-images.githubusercontent.com/51320851/158995916-eac0f2b4-eece-4799-b77c-b8c7a48d4a0e.png)